beautifultable/helpers.py

```diff
@@ -285,10 +285,25 @@ def _get_string(
 
     def __str__(self):
         return self._get_string()
+    
+    def aslist(self):
+        return self.value
+
+    def asdict(self):
+        if self._table.columns.header[0] is None or self._table.columns.header[0] == '':
```

Comment:
we should check this for all headers not just the first one. Also that check could be placed in the below loop where you are reading the values and just return early in case of invalid data.